tox.ini

```diff
@@ -1,14 +1,17 @@
 [tox]
-envlist = linters, py3
+envlist = linters, py3, insync
 isolated_build = True
 
 [testenv]
 deps = poetry
-
-[testenv:linters]
 basepython = python3
 commands=
     poetry install
+    poetry env info
+
+[testenv:linters]
+commands=
+    {[testenv]commands}
     poetry run flake8 --version
     poetry run flake8 setup.py docs ansible_builder test
```

Comment:
It looks like the conflict is with https://github.com/ansible/ansible-builder/pull/3/files#diff-b91f3d5bd63fcd17221b267e851608e8L24, is the intent to revert that and then merge this?

Reply:
Which conflict ? I am not following.

Reply:
Dunno how this happened. Thanks for pointing it out. resolved.